### DISCUSSION TITLE
Erro ao instalar o npm ng-toast curso Angula petShopp

### DISCUSSION THREAD

**Discussion by eduallochio** [balta-io/forum#140] [Tira Dúvidas] [answered]
Boa noite, estou tendo problamas para instalar o ng-toastr, já tenteia a versão que esta no curso que e a  ngx-toastr@10.0.2, e também a versão mais recente que a 13.3.1. Alguém mais passou por esse problema.

up to date, audited 1021 packages in 7s

144 packages are looking for funding
  run `npm fund` for details

5 vulnerabilities (3 moderate, 2 high)

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @eduallochio, qual erro você teve?

**Reply by eduallochio** [reply]
PS C:\dev\petshop> npm audit
# npm audit report

engine.io  4.0.0 - 6.2.0
Severity: moderate
Uncaught exception in engine.io  - https://github.com/advisories/GHSA-r7qp-cfhv-p84w
fix available via `npm audit fix`
node_modules/engine.io

loader-utils  2.0.0 - 2.0.3 || 3.0.0 - 3.2.0
Severity: high
loader-utils is vulnerable to Regular Expression Denial of Service (ReDoS) via url variable - https://github.com/advisories/GHSA-3rfm-jhwj-7488
loader-utils is vulnerable to Regular Expression Denial of Service (ReDoS) via url variable - https://github.com/advisories/GHSA-3rfm-jhwj-7488
loader-utils is vulnerable to Regular Expression Denial of Service (ReDoS) - https://github.com/advisories/GHSA-hhq3-ff78-jv3g
loader-utils is vulnerable to Regular Expression Denial of Service (ReDoS) - https://github.com/advisories/GHSA-hhq3-ff78-jv3g
fix available via `npm audit fix`
node_modules/@angular-devkit/build-angular/node_modules/loader-utils
node_modules/loader-utils
  @angular-devkit/build-angular  13.0.0-next.0 - 13.3.9 || 14.0.0-next.0 - 14.2.9 || 15.0.0-next.0 - 15.0.0-rc.5
  Depends on vulnerable versions of loader-utils
  node_modules/@angular-devkit/build-angular

mime  <1.4.1
Severity: moderate
mime Regular Expression Denial of Service when mime lookup performed on untrusted user input - https://github.com/advisories/GHSA-wrvr-8mpx-r7pp
No fix available
node_modules/force/node_modules/mime
  force  *
  Depends on vulnerable versions of mime
  node_modules/force

5 vulnerabilities (3 moderate, 2 high)

To address issues that do not require attention, run:
  npm audit fix

**Reply by andrebaltieri** [reply]
@eduallochio acredito que este seja apenas um log do `npm audit`. Inclusive pode seguir sem usar ele por enquanto, senão ele vai atualizar os pacotes e vai ficar diferente do curso.

**Reply by eduallochio** [reply]
Ok, muito obrigado.
